### DISCUSSION TITLE
How to get to the page github.com ->community->discussions from the main page github.com ?

### DISCUSSION THREAD

**Discussion by jhon65496** [community/community#108052] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

How to get to the page github.com ->community->discussions from the main page github.com?
https://github.com/orgs/community/discussions

**Comment by bonizario** [comment, marked as answer]
I know a way of doing it through the search bar:

1. Click on the search bar at the top right corner of your screen
2. Type `repo:community/community` and hit enter
3. Now, inside the repository, click on the discussions tab 👍🏻 

If this helped you, consider marking my answer as the "Accepted answer" 😃
